share/doc/homebrew/brew-tap.md

```diff
@@ -54,10 +54,10 @@ edavis/emacs
 
 ## Formula duplicate names
 
-If your tap contains a formula that is also present in master, that's fine,
+If your tap contains a formula that is also present in core Homebrew, that's fine,
```

Comment:
Instead, I'd make that a link to [`homebrew/core`](https://github.com/Homebrew/homebrew-core) like so:

```
[`homebrew/core`](https://github.com/Homebrew/homebrew-core)
```

(That's how we tend to reference taps in the documentation IIRC.)